docs/proposals/20220725-managed-kubernetes.md

```diff
@@ -319,7 +319,7 @@ type GCPManagedClusterSpec struct {
 }
 ```
 
-**This is the design pattern used by AKS in CAPZ**. [An example of how ManagedCluster watches ControlPlane in CAPZ.](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/exp/controllers/azuremanagedcluster_controller.go#L165)
+**This is the design pattern used by AKS in CAPZ**. [An example of how ManagedCluster watches ControlPlane in CAPZ.](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/controllers/azuremanagedcluster_controller.go#L71)
```

Comment:
@jackfrancis Seeing as you wrote this originally I think :slightly_smiling_face: - is this the correct line number / reference?

Reply:
Maybe now already: point to the commit hash instead of a branch, this way the line number won't get outdated :-)
```suggestion
**This is the design pattern used by AKS in CAPZ**. [An example of how ManagedCluster watches ControlPlane in CAPZ.](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/5c69b44ed847365525504b242da83b5e5da75e4f/controllers/azuremanagedcluster_controller.go#L71)
```

Reply:
I think if we pin to a specific commit it's fine to link to a specific line. 

If we want to refer to a branch I would want to get rid of the concrete line numeber. Just no way to detect if code changes and the link points to the wrong place.

Reply:
+1 to using permalinks.

Reply:
I think putting the permalink here is a good compromise - thanks everyone! :slightly_smiling_face: